src/main/scala-2.13/io/altoo/akka/serialization/kryo/ScalaVersionSerializers.scala

```diff
@@ -1,11 +1,11 @@
-package com.romix.akka.serialization.kryo
+package io.altoo.akka.serialization.kryo
 
 import com.esotericsoftware.kryo.Kryo
-import com.romix.scala.serialization.kryo._
+import io.altoo.akka.serialization.kryo.serializer.scala.{ScalaCollectionSerializer, ScalaImmutableMapSerializer}
 
 private[kryo] object ScalaVersionSerializers {
   def mapAndSet(kryo: Kryo): Unit = {
-    kryo.addDefaultSerializer(classOf[scala.collection.MapFactory[scala.collection.Map]], classOf[ScalaImmutableMapSerializer])
+    kryo.addDefaultSerializer(classOf[scala.collection.MapFactory[_root_.scala.collection.Map]], classOf[ScalaImmutableMapSerializer])
```

Comment:
maybe better avoid having a scala package as subpackage so we can get rid of _root_ here?
like serializers.scalabase ?

Reply:
leftover - fixed by adding serializers subpackage